packages/taco/src/tdec.ts

```diff
@@ -61,7 +61,7 @@ export const encryptMessage = async (
 export const retrieveAndDecrypt = async (
   provider: ethers.providers.Provider,
   domain: Domain,
-  porterUri: string,
+  porterUris: string[],
```

Comment:
I don't believe the functions in this module are exposed externally as part of the `taco-web` library. Would it be simpler to create the `PorterClient` object in the `taco.ts` module and pass that object in instead of the string array, or does that cause other issues?

Reply:
I'd need help with that, I'm not that familiar with taco-web to answer your questions.

Reply:
Will be addressed vi #560 - https://github.com/nucypher/taco-web/pull/560#discussion_r1711843350.